Q/QD/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+using BinaryBuilder
+
+name = "QD"
+version = v"2.3.22"
+
+# Collection of sources required to build SDPA-QD
+sources = [
+    "https://www.davidhbailey.com/dhbsoftware/qd-2.3.22.tar.gz" =>
+    "30c1ffe46b95a0e9fa91085949ee5fca85f97ff7b41cd5fe79f79bab730206d3",
+    "./bundled",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/qd-2.3.22
+update_configure_scripts
+./configure --enable-shared --enable-fast-install=no  --prefix=$prefix --host=$target --build=x86_64-linux-gnu
+make
+make install
```

Comment:
Need to use something like
```
make install module_ext=mod
```
?

Reply:
Sure, I honestly have no idea what that does but I am certainly willing to give it a try :)

Reply:
There two problems here: `module_ext` is not being set, passing it that way gives a value (I was suggesting `mod`, but I'm not sure what it should be).  The more important problem is that there is no file called `qmodule.*` at all

Reply:
Strange. It was working on Travis with the old binary builder, do you know what might be different here?

Reply:
Reading the log of your builds, it looks like I guessed the extensions correctly:
```
 /usr/bin/install -c -m 644 qdmodule.mod qdext.mod ddmodule.mod ddext.mod '/workspace/destdir/include/qd'
```
Still, no idea what's happening